src/app/admin/about-us/bars-chart/bars-chart.component.ts

```diff
@@ -0,0 +1,88 @@
+import { Component, OnInit } from '@angular/core';
+import { AboutUsService } from '../about-us.service';
+
+@Component({
+  selector: 'app-bars-chart',
+  templateUrl: './bars-chart.component.html',
+  styleUrls: ['./bars-chart.component.scss']
+})
+export class BarsChartComponent implements OnInit {
+
+  constructor(private aboutUsService: AboutUsService) { }
+
+  public chardReady = false;
+
+  public barChartLabels: string[] = [];
+  public barChartType  = 'bar';
+  public barChartLegend = true;
+  public barChartData;
+  public barChartOptions = {
+    scaleShowVerticalLines: true,
+    responsive: true,
+    scales: {
+      yAxes: [{
+        gridLines: {
+          zeroLineColor: 'black',
+          zeroLineWidth: 1
+        },
+        ticks: {
+          min: 0,
+          max: 100
+        },
+        scaleLabel: {
+          display: true,
+          labelString: 'Кількість коммітів на репозиторій'
+        }
+      }],
+      xAxes: [{
+        scaleLabel: {
+          display: true,
+          labelString: `Ім'я користувача на GitHub`
+        }
+      }]
+    }
+  };
+
+  ngOnInit() {
+    this.getUserNames();
+    this.getCommitsAmount();
+  }
+
+  private async getContributorsInfo() {
+    return this.aboutUsService.getContributorsData().toPromise();
+  }
+
+  private async getUserNickNames() {
+    const data: any = await this.getContributorsInfo();
+    const nickNames = data.map(item => {
```

Comment:
It is better to use contractions 
`const data: any = await this.getContributorsInfo();
return data.map(({author}) => author.login)`
or 
`
return this.getContributorsInfo()
   .then((users) => users.map(({author}) => author.login))
`

Reply:
Tried both ways - no one is working

Reply:
Fixed using second suggestion